tests/testthat/test-utils.R

```diff
@@ -1,15 +1,15 @@
 
-test_that("util_replace_null returns NA", {
+test_that("`util_replace_null` returns NA", {
   expect_equal(util_replace_null(NULL), NA)
   expect_equal(util_replace_null(1), 1)
 })
 
-test_that("util_df_to_list is working", {
+test_that("`util_df_to_list` is working", {
   expect_type(util_df_to_list(ggplot2::mpg), "list")
```

Comment:
Nice idea, but we don't want to rely on ggplot package for this, as it's not a core dependency and we should keep the footprint (list of dependencies as small as possible). Can we use one of the dfs in base R or dplyr (which is a core dependency)?

E.g. `iris` or `dplyr::band_instruments` or `dplyr::starters`?